patrick-lab/.eslintignore

```diff
@@ -0,0 +1,3 @@
+**/node_modules/*
```

Comment:
**TODO FOR LOST POINTS**

This is a top-level comment that that you need to ``npm i -S uuid`` in order for your server to start as I got an error with that.